src/instructlab/model/chat.py

```diff
@@ -173,6 +175,12 @@ def is_openai_server_and_serving_model(
     "--temperature",
     cls=clickext.ConfigOption,
 )
+@click.option(
+    "--rag",
+    is_flag=True,
+    cls=clickext.ConfigOption,
+    required=True,  # default from config
+)
```

Comment:
```suggestion
@click.option(
    "--rag",
    "enabled"
    is_flag=True,
    cls=clickext.ConfigOption,
    config_sections="rag"
    required=True,  # default from config
)
```

you will need to add a bool called `enabled` to the `rag` class you added to `_chat` in configuration.py. These click options need to map to a bool in the config with either the name of the flag `--rag` or the optional string you can provide `enabled`"

Reply:
the config sections point to a sub class in the `_chat` class. For this to work without the `config_sections` `rag` would need to be a `bool` inside of `_chat` directly at the top level